src/store/CategoricalDisplayStore.js

```diff
@@ -39,7 +39,7 @@ export default class CategoricalDisplayStore {
         }
     }
     /* isMobile: fill form the ChartStore */
-    @observable isMobile = false;
+    @computed isMobile = false;
```

Comment:
should be `@computed isMobile = this.mainStore.chart.isMobile;`

Then you can remove
```js
        if (this.mainStore) {
            this.isMobile = this.mainStore.chart.isMobile;
        }
```

Reply:
@binarydevteam ah you already fixed it. Good. Just the DrawTools translation strings (refer above) and we are good to go